spec/support/features/session_helper.rb

```diff
@@ -213,5 +213,91 @@ def session_store
       config = Rails.application.config
       config.session_store.new({}, config.session_options)
     end
+
+    def sign_up_and_2fa_loa1_user_who_came_from_sp
```

Comment:
Yep, I knew you would bring this up, but thought I'd get this out for review and then clean it up.